core/pages/Checkout.js

```diff
@@ -62,7 +62,7 @@ export default {
     this.$bus.$on('checkout-after-shippingMethodChanged', this.onAfterShippingMethodChanged)
     this.$bus.$on('checkout-after-validationError', this.focusField)
     if (!this.isThankYouPage) {
-      this.$store.dispatch('cart/load').then(() => {
+      this.$store.dispatch('cart/load', { forceClientState: true }).then(() => {
```

Comment:
shouldn't be changed if default value is `true`
```suggestion
      this.$store.dispatch('cart/load').then(() => {
```